app/templates/components/deduped-hearings-list.hbs

```diff
@@ -0,0 +1,35 @@
+<h5 class="column-header">Hearing Information</h5>
+    <ul class="no-bullet">
+    {{#each dedupedHearings as |hearing|}}
```

Comment:
awesome. Suggestion: To make this `dedupped-hearings-list` component re-usable and portable to the recommendation form, this component can just yield the `deduppedHearings` as a block parameter. 
For example 
```
{{yield (hash dedupedHearings=dedupedHearings)}}
```

And then all this markup below would be specific to the hearing form (hearing/add.hbs):
```
{{deduped-hearings-list as |dedupedHearings|}}
  <h5 class="column-header">Hearing Information</h5>
    <ul class="no-bullet">
   <li class="grid-x">
        <div class="cell shrink small-margin-right">
          {{#if (is-before hearing.dateofpublichearing)}}
            {{fa-icon "calendar" class="light-gray" fixedWidth=true}}
          {{else}}
            {{fa-icon "check" class="blue" fixedWidth=true}}
          {{/if}}
     ....
{{/deduped-hearings-list}}
```

This way, the recommendation form can add little radio boxes and such next to each dedupped hearing. 
This review comment may not be very clear, happy to chat in person.

Reply:
@trbmcginnis actually it would just be {{yield dedupedHearings}}, my bad

Reply:
will do this in another issue